service/storage_service.cc

```diff
@@ -930,6 +932,8 @@ void storage_service::bootstrap() {
             { gms::application_state::CDC_STREAMS_TIMESTAMP, versioned_value::cdc_streams_timestamp(_cdc_streams_ts) },
             { gms::application_state::STATUS, versioned_value::hibernate(true) },
         }).get();
+        sleep_abortable(std::chrono::seconds(30), _abort_source).get();
```

Comment:
Why did you add this 30s wait?

Reply:
It is a temporary debug patch sneaked in during rebase. I will drop it.